gatsby-config.js

```diff
@@ -112,6 +112,27 @@ const plugins = [
       },
     },
   },
+  {
+    resolve: "gatsby-plugin-git-clone",
+    options: {
+      repository: "https://github.com/markkelnar/wp-graphql.git",
```

Comment:
Pull docs from here for this demo until merged in. If /docs/ doesn't exist, the gatsby-source-filesystem below here will fail.

Reply:
What if while we are transitioning the documentation to github and merging them, we pull in 'develop' branch from the wp-graphql repo?  And once those are merged to master or a release, we can update to use from there?

Reply:
@markkelnar I like the idea of pulling from develop, or even a specific `/docs` branch as we work on the migration.

Reply:
We can make this happen.